imagersite/templates/imagersite/home_page_splash.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'imagersite/base.html' %}
+
+{% load static %}
+
+{% block component_css %}
+<link rel="stylesheet" href="{% static 'css/splash_page.css' %}">
+{% endblock %}
+
+{% block nav %}
+  <nav class="navbar navbar-dark navbar-fixed-top">
+    <button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#collapseEx"><i class="fa fa-bars"></i></button>
+      <div class="container">
+          <div class="collapse navbar-toggleable-xs" id="collapseEx">
+              <a class="navbar-brand" href="/" target="_blank">PictureCrap</a>
+              <ul class="nav navbar-nav">
+                  <li class="nav-item">
+                      <a class="nav-link" href="#home">Home <span class="sr-only">(current)</span></a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#best-features">Features</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#examples-of-use">Examples</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#testimonials">Testimonials</a>
+                  </li>
+              </ul>
+              {% if user.is_authenticated %}
+                <ul class="nav navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="user_profile">Welcome {{ user.username }} </a>
+                    </li>
+                </ul>
+              {% else %}
+                 <ul class="nav navbar-nav">>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/accounts/register/">Register</a>
+                    </li>
+                </ul>
+              {% endif %}
+          </div>
+      </div>
+  </nav>
+{% endblock %}
+
+{% block splash %}
+<script>new WOW().init();</script>
+<div class="view hm-black-strong">
+    <div class="full-bg-img flex-center">
+        <ul>
+            <li>
+                <h1 class="h1-responsive wow fadeInUp">Welcome to Pricture Crap</h1></li>
+            <li>
+                <p class="wow fadeInUp" data-wow-delay="0.2s">It's the shit.</p>
+            </li>
+            <li class="wow fadeInUp" data-wow-delay="0.4s">
```

Comment:
I imagine that if a user is already logged in, they probably would not want to see messages about signing up or logging in. Also that login button probably needs a link.